src/main/java/org/apache/commons/imaging/formats/jpeg/JpegImagingParameters.java

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *  under the License.
+ */
+
+package org.apache.commons.imaging.formats.jpeg;
+
+import org.apache.commons.imaging.formats.tiff.TiffImagingParameters;
+
+/**
+ * Jpeg format parameters.
+ * @since 1.0-alpha3
+ */
+public class JpegImagingParameters extends TiffImagingParameters {}
```

Comment:
Do all the properties on `TiffImagingParameters` apply to JPEGs? (ex, `setExif(TiffOutputSet exif)`)

Reply:
At the moment, due to how the classes are structured, I believe they do. But I've been thinking about it since you first noticed that it was strange the TIFF & JPEG parsers' relationship. Let me see how the code would look like separating the two (with either an intermediary class playing the role of a mixin, or simply duplicating the code…)

Reply:
Done !